src/ios/CDVInAppBrowser.m

```diff
@@ -783,6 +793,16 @@ - (void)showToolBar:(BOOL)show : (NSString *) toolbarPosition
 
 - (void)viewDidLoad
 {
+    CGRect frame = [UIApplication sharedApplication].statusBarFrame;
+    UIToolbar* bgToolbar = [[UIToolbar alloc] initWithFrame:frame];
+    bgToolbar.barStyle = UIBarStyleDefault;
+    bgToolbar.tintColor = [UIColor whiteColor];
+    bgToolbar.barTintColor = [UIColor whiteColor];
+//    [[UIBarItem appearance] setTitleTextAttributes:@{UITextAttributeTextColor : [UIColor whiteColor]}];
+
+
+    [self.view addSubview:bgToolbar];
+    
```

Comment:
I honestly don't know. That's why I mentioned that this merge request somehow contains more stuff and I'm happy to create a 'clean' merge request.